### ISSUE TITLE
Works like charm!

### ISSUE THREAD

**Issue by James-F2** [ledoge/novideo_srgb#5]
Novideo_SRGB and DWM_LUT are god sent, thank you very much ledoge!

The real question is why NVIDIA are denying us this feature when it already exists in the driver? 
The monitor manufacturer has to provide accurate EDID xy gamut coordinates for this to be reliable, maybe that's why NVIDIA is reluctant to enable this in the Control Panel?

Measured with i1 Display Pro
![Gigabyte M32Q](https://user-images.githubusercontent.com/31185814/133754249-608d46b9-276f-48fe-87e0-b0b508852783.jpg)

**Disabled:**
![Disabled](https://user-images.githubusercontent.com/31185814/133754288-1d094f66-1511-4421-a15c-af92375cc40f.png)

**Enabled:**
![Enabled](https://user-images.githubusercontent.com/31185814/133754327-38b473b2-7a07-4c14-ae9e-e5c77fbe7db2.png)

**Comment by James-F2** [comment]
Next step is to let us manually enter measured RGBW xy coordinates, while still preventing idiots from fucking up. :wink:
We don't want "HELP! MY SCREEN IS COMPLETELY BACK, I'M TYPING FROM PHONE" kind of situation. 😛

**Comment by ledoge** [comment]
Good point, definitely need to sanity check custom values – I think it would make more sense to implement it as a system-wide EDID override via the registry though, as opposed to just using different coordinates in the math for this tool. That way it'll work for stuff like AMD's implementation and the color space conversion done by Windows in HDR mode as well.
